usbmisc_imx: allow autoloading on according to dt ids

[ not yet sent upstream ]

Allow udev to autoload the module when booting with device-tree

Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>

Index: linux/drivers/usb/chipidea/usbmisc_imx.c
===================================================================
--- linux.orig/drivers/usb/chipidea/usbmisc_imx.c	2013-05-16 00:46:35.000000000 +0200
+++ linux/drivers/usb/chipidea/usbmisc_imx.c	2013-05-16 09:00:12.000000000 +0200
@@ -175,6 +175,7 @@ static const struct of_device_id usbmisc
 	},
 	{ /* sentinel */ }
 };
+MODULE_DEVICE_TABLE(of, usbmisc_imx_dt_ids);
 
 static int usbmisc_imx_probe(struct platform_device *pdev)
 {
